lib/at_host/at_host.c

```diff
@@ -48,10 +48,7 @@ static struct k_work cmd_send_work;
 
 static inline void write_uart_string(const char *str)
 {
-	/* Send characters until, but not including, null */
-	for (size_t i = 0; str[i]; i++) {
-		uart_poll_out(uart_dev, str[i]);
-	}
+	LOG_RAW("%s", str);
```

Comment:
This can be moved into `response_handler` and `write_uart_string` can be deleted?

Reply:
`write_uart_string` is also called in `cmd_send`